kotsadm/web/src/components/troubleshoot/UploadSupportBundleModal.jsx

```diff
@@ -32,13 +33,18 @@ class UploadSupportBundleModal extends React.Component {
       });
       const analyzedBundle = await response.json();
 
-      this.setState({ fileUploading: false });
-      if (this.props.onBundleUploaded) {
-        this.props.onBundleUploaded(analyzedBundle.id);
+      if (!response.ok) {
+        this.setState({ fileUploading: false, uploadBundleErrMsg: `Unable to upload the bundle: ${analyzedBundle.error}` });
```

Comment:
we should use status code in case of `!response.ok` instead of `analyzedBundle.error`